GDJS/Runtime/pixi-renderers/runtimescene-pixi-renderer.js

```diff
@@ -85,3 +85,11 @@ gdjs.RuntimeScenePixiRenderer.prototype.showCursor = function() {
 gdjs.RuntimeScenePixiRenderer.prototype.getPIXIContainer = function() {
   return this._pixiContainer;
 };
+
+gdjs.RuntimeScenePixiRenderer.prototype.getPIXIRenderer = function() {
+  return this._pixiRenderer;
+}
+
+gdjs.RuntimeScenePixiRenderer.prototype.getScene = function () {
```

Comment:
Interesting, why would you need `getScene`? I assume that code calling this would already know the scene?

Reply:
Uhmm, yes this is not required anywhere at all right now :) I was using this in the `LayerPixiRenderer` (since it passes `RuntimeSceneRenderer`) to query the layers. I believe this was before we added properties in Layer.h and afterward, the code changed a lot. So, I'll be removing this since it's not needed anymore :)